src/cpp/src/visual_language/qwen2vl/classes.cpp

```diff
@@ -292,6 +301,125 @@ ov::Tensor merge_text_and_image_embeddings(
     
 } // namespace qwen2vl_utils
 
+std::shared_ptr<ov::Model> VisionEncoderQwen2VL::patch_preprocess_into_model(std::shared_ptr<ov::Model> model_org) {
```

Comment:
Can you share perf measurement? I'm curious :)

Reply:
qwen2-vl-2b-instruct INT4 default
Input: 8pictures (448x448) + 1K prompt	, output 1K 		
Around 1.4s improvement with this PR, it is quite significant VS the final number 1.8s 
![image](https://github.com/user-attachments/assets/1af4a268-62b0-43bf-95f9-9011b6fc0841)

Reply:
Another case is 1picture(640x480) + 32prompt, there is 3x improvement(3s ->1s).  qwen2.5-vl-3b-instruct INT4 default on LNL.